### PR TITLE
Fix preloading of files from code fixes on Windows

### PR DESCRIPTION
**Pull request by tomv564**
For #370, where getConfiguration now takes windows paths.
The paths from typescript need to be converted from forward slashes back to windows paths.
Small fix, do you want a test or discuss normalised paths?

### CODE REVIEW

File: src/typescript-service.ts [hunk 1]
```diff
@@ -1245,7 +1245,9 @@ export class TypeScriptService {
 
         return this.projectManager.ensureOwnFiles(span)
             .concat(Observable.defer(() => {
-                const configuration = this.projectManager.getConfiguration(fileTextChanges[0].fileName)
+                // Configuration lookup uses Windows paths, FileTextChanges uses unix paths. Convert to backslashes.
+                const firstChangedFile = uri2path(path2uri(fileTextChanges[0].fileName))
```
Comment (felixfbecker) [inline src/typescript-service.ts]: If the purpose of this is to convert to backslashes, why not simply do `filename.replace(/\//g, '\\')`? Converting to a URI and back seems like a hack

Comment (felixfbecker) [reply src/typescript-service.ts]: I see that the intention is to only do it if it's a path with a drive letter, I'd still prefer to leave the URI functions out of this

Comment (tomv564) [reply src/typescript-service.ts]: Yes it's not the cleanest looking solution :) I'll extract the needed logic from uri2path.

Comment (tomv564) [reply src/typescript-service.ts]: Done.

**Pull request merged**